core/Plugin/Segment.php

```diff
@@ -296,6 +298,32 @@ public function setSuggestedValuesCallback($suggestedValuesCallback)
         $this->suggestedValuesCallback = $suggestedValuesCallback;
     }
 
+    /**
+     * @return string
+     * @ignore
+     */
+    public function getSuggestedValuesApi()
+    {
+        return $this->suggestedValuesApi;
+    }
+
+    /**
+     * Set callback which will be executed when user will call for suggested values for segment.
+     *
+     * @param string $suggestedValuesApi
+     */
+    public function setSuggestedValuesApi($suggestedValuesApi)
+    {
+        if (!empty($suggestedValuesApi) && is_string($suggestedValuesApi)) {
+            if (Development::isEnabled() && strpos($suggestedValuesApi, '.get') === false) {
```

Comment:
Guess we can ignore that this might throw a warning in development mode if `setSuggestedValuesApi` is called with something else than a string